src/main/java/com/red5pro/cluster/streams/ProfileLevelResolver.java

```diff
@@ -0,0 +1,41 @@
+package com.red5pro.cluster.streams;
+
+import java.util.List;
+import java.util.concurrent.CopyOnWriteArrayList;
+/**
+ * @author Andy Shaules
+ *
+ */
+public interface ProfileLevelResolver {
+	/**
+	 * used to determine the priority level of a multi bitrate stream. Parses the stream name to extract the bitrate priority. 
+	 */
+	public static List<ProfileLevelResolver> LevelResolvers = new CopyOnWriteArrayList<>();
+	/**
```

Comment:
I'd prefer a CopyOnWriteArraySet here to prevent duplicates; this would also require that hashCode and equals for the PLR impl return appropriate values for the instances.